appinventor/blocklyeditor/src/field_lexical_variable.js

```diff
@@ -852,6 +902,14 @@ Blockly.LexicalVariable.makeLegalIdentifier = function(ident) {
   } else if (check.transformed === '') {
     return '_';
   } else {
+    // TODO: Is there a good translated thing we could default this to?
```

Comment:
Given that this is always indirectly called from a FieldFlydown, we could probably change the intermediate signatures to take both `newName` and `defaultName`, which would be the initial text--which is internationalized--for the flydown set by the corresponding parent block's `init()` method.

Reply:
Sounds like a good plan. I think I might separate this out into a different issue though. Looking back at all of these todos made me thing I may have gone a bit change-happy haha. I want to try and stay focused on making loops easier to work with.